code/game/objects/structures/rappel_system.dm

```diff
@@ -285,8 +285,8 @@
 /obj/structure/dropship_equipment/shuttle/rappel_system/on_launch()
 	if(rappel_condition == RAPPEL_CONDITION_HOOKED) //Tadpole is moving with ropes hooked, so the ropes will snap
 		var/turf/target_floor = get_turf(rope)
-		target_floor.balloon_alert_to_viewers("As the dropship tries to leave, the rappel cord is ripped out from above, disabling it!")
-		balloon_alert_to_viewers("The force of the tadpole pulls on the rappel cord, causing a horrible screeching sound as something breaks!")
+		target_floor.balloon_alert_to_viewers("the rope is ripped out from above!")
+		visible_message(span_boldwarning("The rope of \the [src] is ripped out from under the dropship, making a horrible screeching sound!"))
```

Comment:
some of these like this one I don't really agree with.
The balloon alert means anyone on screen will actually SEE it where as text messages are easily missed, especially when there is a lot going on.
IMO visible messages are best where its useful to know after the fact what has happened. In a case like this, its useful to know immediately what has happened (i.e. I can't use the rappel anymore/I can move the tad immediately), where as 60 seconds later it doesn't matter, as there is visual feedback that the rappel is broken.